src/LibSingular.jl

```diff
@@ -4,8 +4,14 @@ import Libdl
 using CxxWrap
 @wrapmodule(realpath(joinpath(@__DIR__, "..", "local", "lib", "libsingularwrap." * Libdl.dlext)))
 
+mapping_types = nothing
+
 function __init__()
    @initcxx
+   global mapping_types, casting_functions
+   mapping_types = get_type_mapper()
```

Comment:
Really? :-) I'd at least expect consistency between the variable and the getter, e.g. `type_mapper = get_type_mapper()` or `mapping_types = get_mapping_types`.